test/new-e2e/tests/agent-subcommands/flare/flare_win_test.go

```diff
@@ -9,24 +9,24 @@ package flare
 import (
 	"testing"
 
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2os"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2params"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/host"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client/agentclient"
+	"github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+
 	"github.com/stretchr/testify/assert"
 )
 
-type windowsFlareSuite struct {
-	baseFlareSuite
-}
+type windowsFlareSuite baseFlareSuite
 
 func TestWindowsFlareSuite(t *testing.T) {
-	e2e.Run(t, &windowsFlareSuite{}, e2e.FakeIntakeStackDef(e2e.WithVMParams(ec2params.WithOS(ec2os.WindowsOS))))
+	t.Parallel()
```

Comment:
I think we already run tests in different jobs, I would rather avoid this unexpected change